ods_ci/tests/Tests/400__ods_dashboard/410__ods_dashboard_projects/410__ods_dashboard_projects_edit.robot

```diff
@@ -105,17 +105,16 @@ Project Suite Setup
     Set Library Search Order    SeleniumLibrary
     RHOSi Setup
     Launch Data Science Project Main Page
+    Delete Data Science Projects From CLI    ${PROJECTS_TO_DELETE}
     Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
-    ...    resource_name=${PRJ_RESOURCE_NAME}
+    ...    resource_name=${PRJ_RESOURCE_NAME}    existing_project=${TRUE}
```

Comment:
is it because the project is always already present (e.g., Suite/Test Setup)? or just due to sequential run of the tests?

Reply:
Due to multiple runs, allowing the reuse of the project saves time

Reply:
what do you mean by runs? Different jobs?

Reply:
Multiple execution of ODS-CI script, either via Jenkins or locally.
The option `existing_project=${TRUE}` is commonly used in our tests, and I made it to reduce time if a test can reuse an existing project.

Reply:
hm yeah I remember that for re-using in the same run.

I-d argue that it is a good behavior to re-use over multiple ods-ci runs. I think every run should find the cluster clean from previous execution otherwise it would miss covering important pieces like project creation

Reply:
Our tests do not always use `existing_project=${TRUE}`. However, in some test cases it is more efficient to reuse the project.